openapi/components/schemas/User.yaml

```diff
@@ -52,6 +52,10 @@ properties:
     description: Either a date-time or empty string.
   last_platform:
     $ref: ./Platform.yaml
+  last_mobile:
+    type: string
+    format: date-time
+    description: Either a date-time or empty string.
```

Comment:
`last_mobile` is not present in the response:
https://github.com/vrchatapi/specification-test/blob/main/data/requests/users/getuser.md

Reply:
Upon further investigation, I discovered and successfully tested that the field is only accessible if the authenticated account is friends with the target account.